src/main/scala/org/broadinstitute/dsde/workbench/leonardo/Boot.scala

```diff
@@ -68,6 +71,10 @@ object Boot extends App with LazyLogging {
       dbRef.database.close()
     }
 
+    val petGoogleDAO: (WorkbenchEmail, GoogleProject) => GoogleStorageDAO = (email, project) => {
+      new HttpGoogleStorageDAO(dataprocConfig.applicationName, Token(() => serviceAccountProvider.getAccessToken(email, project)), "google")
```

Comment:
I guess it'd be nice if `Token` took a `(WorkbenchEmail, GoogleProject) => String` instead of a `() => String`. Then we'd be able to use the same `HttpGoogleStorageDAO` instance instead of constructing a new one each time. Not a big deal for this PR though.